server/app/controllers/HomeController.java

```diff
@@ -62,6 +67,12 @@ public CompletionStage<Result> index(Http.Request request) {
                 // If the applicant has not yet set their preferred language, redirect to
                 // the information controller to ask for preferred language.
                 ApplicantData data = applicant.getApplicantData();
+                if (langs.availables().size() <= 1 && !data.hasPreferredLocale()) {
```

Comment:
Hm, I think it worth a comment here saying that we are setting preferred locale when a single language is available so that step of language selection can be automatically skipped. Otherwise it's a bit unclear why is it useful to "choose" locale for user.